### PR TITLE
multi_disk: Add iozone stress step on random disks

### PR DESCRIPTION
**Pull request by YongxueHong**
Do iozone stress on the random data disks.

ID: 1572520
Signed-off-by: Yongxue Hong <yhong@redhat.com>

### CODE REVIEW

File: qemu/tests/multi_disk.py [hunk 1]
```diff
@@ -268,9 +272,12 @@ def _do_post_cmd(session):
         _do_post_cmd(session)
         raise
     try:
+        if iozone_options:
+            iozone = generate_instance(params, session, 'iozone')
+            random.shuffle(disks)
```
Comment (hereischen) [inline qemu/tests/multi_disk.py]: You can use `random.sample(population, k)`

**Comment by hereischen** [comment]
Acked-by: Haotong Chen hachen@redhat.com

**Comment by CongLi** [comment]
Could you please explain why 4~5 disks instead of max disks on different block drivers?

**Comment by YongxueHong** [comment]
@CongLi   it's not instead of max disks, just add a step which execute iozone stress on 4~5 disks for adding a new case named "Running iozone on each guest when start VM w/ 29 virtual disk". The details you could see BZ.

**Comment by CongLi** [comment]
Ack.

**Comment by YongxueHong** [comment]
@XueqiangWei  @ngu-niny  @peixiu @kimi1978wy  if you are free, pls help me review, thanks.

**Comment by YongxueHong** [comment]
@XueqiangWei  @ngu-niny  @peixiu @kimi1978wy  if you are free, pls help me review, thanks.

**Comment by kimi1978wy** [comment]
LGTM and tested on 2012r2 with blk device, passed:
 (1/1) Host_RHEL.m7.u7.qcow2.virtio_blk.up.virtio_net.Guest.Win2012.x86_64.r2.io-github-autotest-qemu.multi_disk.max_disk: PASS (559.86 s)

Acked-by: Yu Wang <wyu@redhat.com>

**Comment by ngu-niny** [comment]
Test PASS:

3  Host_RHEL.m8.u0.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.1.0.ppc64le.io-github-autotest-qemu.multi_disk.max_disk PASS  5752.50s

Acked-by: Nini Gu <ngu@redhat.com>

**Comment by YongxueHong** [comment]
Thanks all.

**Pull request merged**